operator/controllers/seldondeployment_prepackaged_servers.go

```diff
@@ -129,9 +133,10 @@ func (pi *PrePackedInitialiser) addTFServerContainer(mlDepSpec *machinelearningv
 	}
 
 	envSecretRefName := extractEnvSecretRefName(pu)
+	storageInitializerImage := extractStorageInitializerImage(pu)
```

Comment:
why do we need a separate function for this as it just returns the value from `pu`